src/spec/Paths.php

```diff
@@ -31,7 +31,7 @@
 class Paths implements SpecObjectInterface, DocumentContextInterface, ArrayAccess, Countable, IteratorAggregate
 {
     /**
-     * @var PathItem[]
+     * @var (PathItem|null)[]
```

Comment:
not sure if I understand this correctly. But this looks like it could be an array of nulls? does not make much sense to me.

Reply:
Look at line 53. The `$_paths` array can contain `null`s as it's items.